client/src/components/StudentCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import StudentCardCSS from "../css/StudentCard.module.css";
+import { useNavigate } from "react-router-dom";
+
+const StudentCard = () => {
+
+    const navigate = useNavigate();
+
+    let studentData = {
+        name: "Jon Snow",
+        email: "jonsnow@example.com",
+        phone: "555-123-4567",
+        cohort: "MCSP-35",
+        status: "Technical Interview",
+        createdBy: "Rane Gray",
+        interviewDate: "June 6th, 2023 @ 10:00AM MST",
+        notes: {
+            attemptNum: 1,
+            date: "May 23rd, 2023",
+            narrative: "Jon performed decently on most topics. Struggled with accessing properties on an object"
+        }
+    }
+
+    var statusStyle = "";
+
+    if(studentData.status == "Technical Interview"){
+        statusStyle = StudentCardCSS.interview;
+    }
+    if(studentData.status == "Coding Challenge"){
+        statusStyle = StudentCardCSS.challenge;
+    }
+    if(studentData.status == "Prep Work"){
+        statusStyle = StudentCardCSS.prep;
+    }
+    if(studentData.status == "Done"){
+        statusStyle = StudentCardCSS.done;
+    }
```

Comment:
What if we made a status tag component that we can use on the card and in the table.

We can move this logic into that and all the CSS classes from the module to a new module for better reusability.

Let me know if you have any questions.